Lib/test/test_genexps.py

```diff
@@ -153,7 +153,7 @@
     >>> (y for y in (1,2)) = 10
     Traceback (most recent call last):
        ...
-    SyntaxError: cannot assign to generator expression
+    SyntaxError: cannot assign to generator expression. Maybe you meant '==' instead of '='?
```

Comment:
Again, I'm not convinced that the suggestion is very helpful; I cannot imagine what went wrong but `==` is not my first guess.

Reply:
Yeah, in this case I agree. Not sure if we can easily exclude generators, though. I can try to investigate this in a future PR

Reply:
Actually, it was quite easy to exclude generators